pkg/apis/core/v1beta1/helper/helper_suite_test.go

```diff
@@ -17,11 +17,13 @@ package helper_test
 import (
 	"testing"
 
+	"github.com/gardener/gardener/pkg/gardenlet/features"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
 func TestHelper(t *testing.T) {
+	features.RegisterFeatureGates()
```

Comment:
Remove this from here (gardenlet feature gates should not be in the API package)

Reply:
Removed.